middleware-catalog-module/catalog-dao/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst

```diff
@@ -1,3 +0,0 @@
-co\tpg\catalog\dao\exception\BackendException.class
```

Comment:
All classes on directory /target must be in .ignore. Now we need to delete them to remove from repository. After deleting do git add, git commit, git push.